projects/kenlm/kenlm_fuzzer.cc

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 Google Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <iostream>
+#include <string>
+#include "lm/common/model_buffer.hh"
+#include "lm/model.hh"
+#include "lm/state.hh"
+
+
+extern "C" int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  char filename[256];
+  sprintf(filename, "/tmp/libfuzzer");
```

Comment:
Can I ask why we're making a filename variable only to put a constant in it?  Or is this just a draft in progress?

Reply:
Did a bit of refactoring. If you still have comments, feel free to reopen the thread.